builder/dockerfile/containerbackend.go

```diff
@@ -0,0 +1,156 @@
+package dockerfile
```

Comment:
I meant the `fmt.Fprintf()` stuff, but I think it's fine if it's in `api/server/backend/build`